generic/OpenMP_Support.F90

```diff
@@ -570,6 +576,67 @@ subroutine set_services(gc, rc)
             end do
            _RETURN(ESMF_SUCCESS)
         end subroutine set_services
+
     end function make_subgridcomps
 
+    subroutine copy_callbacks(state, multi_states, rc)
+       use mapl_ESMF_Interfaces
+       use mapl_CallbackMap
+       type(ESMF_State), intent(inout) :: state
+       type(ESMF_State), intent(inout) :: multi_states(:)
+       integer, optional, intent(out) :: rc
+
+       integer :: n_multi, i
+       integer :: status
+       type(CallbackMethodWrapper), pointer :: wrapper
+       type(CallbackMap), pointer :: callbacks
+       type(CallbackMapIterator) :: iter
+
+       n_multi = size(multi_states)
+       call get_callbacks(state, callbacks, _RC)
+       if (associated(callbacks)) then
```

Comment:
There should be an assertion that callbacks is associated.  Even if the dictionary is empty, we should have a pointer to a valid (albeit empty) dictionary.

Reply:
Will do!